### ISSUE TITLE
SST DEV results in :443 connection refused

### ISSUE THREAD

**Issue by uripg** [anomalyco/sst#5184]
Hello!
I'm trying to run the barebones [hono serverless](https://sst.dev/docs/start/aws/hono).

Everything works just fine up until the moment I run 
```npx sst dev```
which crashes.

```
time=2024-12-06T13:53:43.750+01:00 level=INFO msg="checking for pulumi" path="/Users/uri/Library/Application Support/sst/bin/pulumi"
time=2024-12-06T13:53:44.096+01:00 level=INFO msg="checking for bun" path="/Users/uri/Library/Application Support/sst/bin/bun"
time=2024-12-06T13:53:44.107+01:00 level=INFO msg="initializing project" version=3.3.50
time=2024-12-06T13:53:44.108+01:00 level=INFO msg="esbuild building" out=/Users/uri/Documents/Repositories/aws-hono/.sst/platform/sst.config.1733489624108.mjs
time=2024-12-06T13:53:44.116+01:00 level=INFO msg="esbuild built" outfile=/Users/uri/Documents/Repositories/aws-hono/.sst/platform/sst.config.1733489624108.mjs
time=2024-12-06T13:53:44.116+01:00 level=INFO msg="evaluating config"
time=2024-12-06T13:53:44.165+01:00 level=INFO msg="config evaluated"
time=2024-12-06T13:53:44.165+01:00 level=INFO msg="checking platform"
time=2024-12-06T13:53:44.166+01:00 level=INFO msg="installing platform"
⠋  Upgrading project...time=2024-12-06T13:53:44.234+01:00 level=INFO msg="installing deps"
time=2024-12-06T13:53:44.234+01:00 level=INFO msg="getting package" name=@sst-provider/aws version=latest
⠧  Installing providers...time=2024-12-06T13:53:44.914+01:00 level=INFO msg="getting package" name=@pulumi/aws version=latest
⠸  Installing providers...time=2024-12-06T13:53:45.564+01:00 level=INFO msg="writing package.json"
time=2024-12-06T13:53:45.565+01:00 level=INFO msg="adding dependency" name=aws
time=2024-12-06T13:53:45.567+01:00 level=INFO msg="fetching deps"
⠸  Installing providers...time=2024-12-06T13:53:49.539+01:00 level=INFO msg="writing types"
time=2024-12-06T13:53:49.539+01:00 level=INFO msg="loading home"
time=2024-12-06T13:53:49.541+01:00 level=INFO msg="aws credentials found" region=eu-west-1 profile=""
time=2024-12-06T13:53:49.541+01:00 level=INFO msg="fetching bootstrap"
⠏  Installing providers...time=2024-12-06T13:53:50.213+01:00 level=INFO msg="found existing bootstrap" data="{\"version\":5,\"asset\":\"sst-asset-tmafrehsnmvr\",\"assetEcrRegistryId\":\"717279690149\",\"assetEcrUrl\":\"717279690149.dkr.ecr.eu-west-1.amazonaws.com/sst-asset\",\"state\":\"sst-state-tmafrehsnmvr\",\"appsyncHttp\":\"\",\"appsyncRealtime\":\"\"}"
time=2024-12-06T13:53:50.213+01:00 level=INFO msg="loaded config" app=aws-hono stage=uri
time=2024-12-06T13:53:50.213+01:00 level=INFO msg=mosaic project=/Users/uri/Documents/Repositories/aws-hono
time=2024-12-06T13:53:50.214+01:00 level=INFO msg="starting watcher" root=/Users/uri/Documents/Repositories/aws-hono
time=2024-12-06T13:53:50.215+01:00 level=INFO msg="server port assigned" port=13557
time=2024-12-06T13:53:50.215+01:00 level=INFO msg="getting endpoint"
time=2024-12-06T13:53:50.216+01:00 level=INFO msg=watching path=/Users/uri/Documents/Repositories/aws-hono
time=2024-12-06T13:53:50.217+01:00 level=INFO msg=watching path=/Users/uri/Documents/Repositories/aws-hono/src
time=2024-12-06T13:53:50.217+01:00 level=INFO msg=server addr=0.0.0.0:13557
time=2024-12-06T13:53:50.218+01:00 level=INFO msg="shutting down server"
time=2024-12-06T13:53:50.218+01:00 level=INFO msg="server done"
time=2024-12-06T13:53:50.217+01:00 level=INFO msg="watcher done"
time=2024-12-06T13:53:50.221+01:00 level=INFO msg="done mosaic" err="dial tcp :443: connect: connection refused"
```

```time=2024-12-06T13:53:50.221+01:00 level=ERROR msg="exited with error" err="dial tcp :443: connect: connection refused"```

I've tried different computers and networks so I assume its an issue with my aws config? But I'm not sure how to debug this further.

Thanks!

**Comment by uripg** [comment]
Gone keep it open as this is not a true fix but for now

downgrading to "3.2.16" instead of the "3.3.50" that I was using makes it work just fine!

**Comment by thdxr** [comment]
this should be fixed in v3.3.55
